tests/test-options.ts

```diff
@@ -0,0 +1,98 @@
+import { test as base } from '@playwright/test';
+import { PageManager } from '../pages/pageManager';
+import { RandomDataHelper } from '../pages/randomDataHelper'
+import { APIHelper} from '../pages/apiHelper'
+
+
+export type TestOptions = {
+    pageManager: PageManager
+    newOwnerFixture: [{
+        'firstName': string,
+        'lastName': string,
+        'address': string,
+        'city': string,
+        'telephone': string,
+        'petName': string,
+        'petBirthdate': Date,
+        'petType': string,
+        'visitDate': Date,
+        'visitDescription': string
+    }]
```

Comment:
The data type not necessarily should be array. In your example its more reasonable to have an object, instead of array of objects. You will have a single set of data per test anyway. Every fixture has an isolated instance per test.

Reply:
Thanks for clearing up this confusion.  I thought it had to be an array for some reason, but I think I was thinking about something else, like fixture options.